Backpack-SwiftUI/Utils/Classes/TwoRowGrid.swift

```diff
@@ -68,11 +68,13 @@ struct TwoRowGrid<Item, ItemView: View>: View {
     
     private func column(proxy: GeometryProxy, columnIndex index: Int, rowIndex: Int) -> some View {
         let itemTuple = items[rowIndex][index]
+        let shape = RoundedRectangle(cornerRadius: .md)
         return itemView(itemTuple.0, itemTuple.1)
             .if(rowIndex % 2 != 0) {
                 $0.frame(width: (proxy.size.width / 2) - (spacing.value / 2))
             }
-            .clipShape(RoundedRectangle(cornerRadius: .md))
+            .clipShape(shape)
+            .contentShape(shape)
```

Comment:
Found that the hit test returned wrong result which leads to swiftui call the wrong `onTapGesture` handler,
here we add `contentShape` to refresh the hit area, so that it'll call the correct event handler.

Reply:
Ahh nice!